README.md

```diff
@@ -83,8 +83,8 @@ Symbol Definitions:
 
 - [x] 2024Q1 - Enable producing the [zkEVM](https://github.com/kroma-network/zkevm-circuits) proof.
 - [ ] 2024Q2 - Replace Halo2 with Tachyon in [Kroma](https://kroma.network/) mainnet.
-- [ ] 2024Q3 - Implement Halo2 GPU.
-- [ ] 2024Q4 - Implement Halo2 Folding Scheme.
+- [ ] 2024Q3 - Implement and optimize Plonky3 in Tachyon.
+- [ ] 2024Q3 - Replace zkEVM with zkVM in [Kroma](https://kroma.network/) mainnet.
```

Comment:
Might be too verbose, but to justify the implementation of Plonky3 in Tachyon and alignment with zkVM, how about specifying zkVM as SP1 zkVM?